server.go

```diff
@@ -295,26 +303,76 @@ type Server struct {
 	DecorateReader DecorateReader
 	// DecorateWriter is optional, allows customization of the process that writes raw DNS messages.
 	DecorateWriter DecorateWriter
-
```

Comment:
Leave the new line seperating public from private.

Reply:
Done